rules/windows/process_creation/proc_creation_win_susp_7zip_dmp.yml

```
@@ -0,0 +1,30 @@
+title: Winrar Compressing Dump Files
```

I think the title doesn't fit the content of rule
```suggestion
title: 7Zip Compressing Dump Files
```